python/graphscope/tests/kubernetes/test_demo_script.py

```diff
@@ -308,6 +328,55 @@ def test_demo_distribute(gs_session_distributed, data_dir, modern_graph_data_dir
     # GNN engine
 
 
+def test_demo_with_lazy_mode(gs_session_with_lazy_mode, data_dir, modern_graph_data_dir):
```

Comment:
Since this lazy focus on the automated pod creating and destroying, could you add the check for the pod? launch -> check the existence and number of specific pod. destroy(close) -> check the nonexistence of the pod.